openedx/core/djangoapps/heartbeat/default_checks.py

```diff
@@ -22,6 +22,12 @@
 # Modulestore
 
 def check_modulestore():
+    """ Check the modulestore connection
+
+    Returns:
+        unicode: A unicode string of either "OK" or the failure message
```

Comment:
That's not what it returns.  It returns a tuple of (service, ok-bool, message).  I'm OK with saying this in one place, and then having these docstrings say, "Returns the tuple expected by blah blah" or something.